### DISCUSSION TITLE
How do I hide behind-the-scene requests in The Logger?

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#2711] [Q&A] [answered]
I probably missed this being documented somewhere. Is this possible?

**Comment by gorhill** [comment]
Not possible.

**Reply by partingscientist** [reply]
Ah that is unfortunate. Thank you regardless.

**Reply by gorhill** [reply, marked as answer]
https://github.com/gorhill/uBlock/commit/5d6e10318662d400c78eb2dd5062a3e1194e387a

**Reply by partingscientist** [reply]
Well that's unexpected. I tested it and it seems to work well. Thank you.
